### ISSUE TITLE
Cannot compile for wasm with simd enabled

### ISSUE THREAD

**Issue by etemesi254** [tirr-c/jxl-oxide#308]
Hi there, thanks for the library.

The library (version `0.8.0` ) can't be compiled for wasm+simd due to the fact that jxl-grid doesn't  have `SimdVector` for `v128` defined in crates.io

Errors look like

```shell
error[E0599]: no method named `store` found for struct `std::arch::wasm32::v128` in the current scope
   --> /home/caleb/.cargo/registry/src/index.crates.io-6f17d22bba15001f/jxl-render-0.8.0/src/vardct/wasm32/transform.rs:118:27
    |
118 |             scratch_1[y2].store(row_ptr.add(4));
    |                           ^^^^^ method not found in `v128`

````

While the methods exist in repository, aka https://github.com/tirr-c/jxl-oxide/blob/f37a3980a9cc2b99f96fded54d1dd41a824571cd/crates/jxl-grid/src/simd.rs#L405-L495, 

they do not exist in `jxl-grid` in crates.io (https://docs.rs/crate/jxl-grid/0.4.1/source/src/simd.rs) causing the errors above

**Comment by tirr-c** [comment]
Oops, I forgot to bump version of jxl-grid... I just published jxl-grid 0.4.2 and jxl-render 0.8.1, and yanked jxl-render 0.8.0. Could you check if it's fixed now?

**Comment by etemesi254** [comment]
Perfect, it's working
